packages/utils/src/fetch/core.ts

```diff
@@ -0,0 +1,62 @@
+import fetch from 'cross-fetch';
+import { requestHandler, RequestOptions, responseHandler } from './handlers';
+
+export type WithoutRequestBodyOptions = Omit<RequestOptions, 'body'>;
+
+export function create(baseUrl: string, options: RequestOptions) {
+  return {
+    doRequest: (path: string) => doRequest(`${baseUrl}/${path}`, options),
```

Comment:
```suggestion
    request: (path: string) => doRequest(`${baseUrl}/${path}`, options),
```
이미 `request`, `get`, `post` 등의 단어들이 이 함수가 하는 일을 잘 나타낸다고 생각이 들어서 `do` prefix 빼는 것도 좋을 것 같은데 어떠신가유?

Reply:
아 이건 delete가 자바스크립에 이미 예약어로 있어서 이렇게 했어요..!
delete를 대체할만한게 있을까요...🤔

Reply:
`http.delete` 처럼 접근하면 `delete`도 사용할 수 있지 않나유????? 👀